serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,51 @@
+= Migrate Your Postgresql database with Kogito upgrade
+
+Kogito provides out of the box way to manage your database schema changes with each upgrade. Kogito uses flyway to achieve the same functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+== How to migrate
+
+.Ways to Migrate your Kogito database with Kogito and Flyway
```

Comment:
I see 3 options to migrate the database, is that right ?
Can we separate them in 3 different subtitles, with explicit names, so they are also accessible via the right menu ?